search-app/src/cmr/search/services/parameters/parameter_validation.clj

```diff
@@ -459,6 +469,23 @@
           []
           months)))))
 
+(defn temporal-facet-day-validation
+  "Validates that the days provided in all temporal-facet parameters are valid."
+  [concept-type params]
+  (when-let [param-values (:temporal-facet params)]
```

Comment:
This parameter value logic is also in multiple places. I imagine splitting it out will be tricky, but worth trying

Reply:
Refactored.